qa/clusters/fixed-2.yaml

```diff
@@ -10,3 +10,5 @@ overrides:
     conf:
       osd:
         osd shutdown pgref assert: true
+    log-ignorelist:
+      - \(POOL_APP_NOT_ENABLED\)
```

Comment:
Why is this warning being ignored in `fixed-2` snippet but not in `fixed-3` or other generic cluster definition snippets in `qa/clusters`?  These snippets are used by many suites, sometimes interchangeably, so it would be nice if they were consistent.

Reply:
@idryomov I have added POOL_APP_NOT_ENABLED to ignorelist in the selected test yamls based on failures observed during rados testsuite run. I do agree that we should add this in most of the generic definitions but afraid that we may miss out on genuine warnings of  POOL_APP_NOT_ENABLED hence added to related tests which are having valid failures. Let me know you thoughts on it.

Reply:
> I do agree that we should add this in most of the generic definitions

This is not what I suggested -- I merely said that all generic definitions should be consistent.  If some warning is ignored in one them, it should be ignored in all of them.

Personally, I think that we shouldn't ignorelist any warnings in generic cluster definition such as `fixed-2` or `fixed-3` so that we don't need to remember that this particular warning is ignored by default and also that there is no precedent for folks to start piling on.  The approach of adding targeted ignorelist entries to tests that need them is much better IMO.

Reply:
Understood. Let me try to add POOL_APP_NOT_ENABLED to ignorelist for relevant tests rather than to generic definitions.

Reply:
@idryomov I have removed the POOL_APP_NOT_ENABLED warning from the selected generic definitions and added it to the relevant tests. Kindly review the recent changes.